documentation-site/pages/components/accordion.mdx

```diff
@@ -50,48 +59,15 @@ Accordions display a list of high-level options that can expand/contract to reve
   <StatefulAccordion />
 </Example>
 
-## Overrides
-
-<Overrides
-  name="Accordion"
-  component={AccordionExports}
-  renderExample={props => {
-    const content = 'Some sample content';
-    return (
-      <Accordion overrides={props.overrides}>
-        <Panel overrides={props.overrides} title="Accordion panel 1">
-          {content}
-        </Panel>
-        <Panel overrides={props.overrides} title="Accordion panel 2">
-          {content}
-        </Panel>
-        <Panel overrides={props.overrides} title="Accordion panel 3">
-          {content}
-        </Panel>
-      </Accordion>
-    );
-  }}
-/>
-
 **Note:** the `Panel` has the same overrides, with the exception of the `Root` subcomponent.
 
 ## API
 
-<API
-  heading="Accordion API"
-  api={require('!!extract-react-types-loader!../../../src/accordion/accordion.js')}
-/>
-
 <API
```

Comment:
Kept this on purpose, because I am not sure how we could include this API in Yard. @tajo thoughts?

Reply:
I would still remove it. I think the only prop that's being ever used is `title` and that's pretty obvious even from the yard example. You will probably never use `Panel` as a standalone thing.

Anyway, this a bit awkward component because it doesn't have stateless version which would be much better fit yard.

Reply:
sounds good